CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
-# Copyright (C) 2017-2020 Fondazione Istituto Italiano di Tecnologia
-# Authors: Silvio Traversaro <silvio.traversaro@iit.it>
-# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LICENSE
+# SPDX-FileCopyrightText: Fondazione Istituto Italiano di Tecnologia (IIT)
+# SPDX-License-Identifier: BSD-3-Clause
 
 cmake_minimum_required(VERSION 3.5)
 project(yarp-device-xsensmt)
```

Comment:
Regarding setting the project version in the CMake file, I see it's done for many other projects, but here it's not set.

I'm not sure if it's needed.

Reply:
We are not installing any CMake version file for the project, so there is no benefit in keeping the version in the CMake project.

Reply:
Makes sense!